chain-signatures/node/src/protocol/cryptography.rs

```diff
@@ -251,11 +232,10 @@ impl CryptographicProtocol for ResharingState {
     }
 }
 
-#[async_trait]
 impl CryptographicProtocol for RunningState {
-    async fn progress<C: CryptographicCtx + Send + Sync>(
-        mut self,
-        ctx: C,
+    async fn progress(
+        self,
+        ctx: &mut MpcSignProtocol,
```

Comment:
We should rename either field or struct name

Reply:
hmm, I'll give it another pass another time